modules/layers/src/mode-handlers/translate-handler.js

```diff
@@ -2,18 +2,182 @@
 
 import turfBearing from '@turf/bearing';
```

Comment:
Move all of this to snap-translate-handler.js

Reply:
Do you want me to rename the class to SnapTranslateHandler also? And on top of that, do you want me to rename the `EditableGeoJsonLayer` default props `modeHandler` key from `translate` to `snapTranslate`?
```js
modeHandlers: {
  snapTranslate: new SnapTranslateHandler()
}
```
 I worry that the latter change will be a breaking change though.